test/vinyl/layout.result

```diff
@@ -185,16 +185,6 @@ result
           type: INSERT
         BODY:
           tuple: [11, {}]
-      - HEADER:
-          timestamp: <timestamp>
-          type: INSERT
-        BODY:
-          tuple: [7, {2: 5}]
-      - HEADER:
-          timestamp: <timestamp>
-          type: INSERT
-        BODY:
-          tuple: [7, {2: 4}]
```

Comment:
@locker I need your help here. I failed to understand the vylog internals myself (though I have read `rfc/vylog-ups-and-downs.md`) and I am not sure whether this change in the layout result is correct or not...

Reply:
This happens because `vy_run_remove_files()` now returns -1 if the index directory isn't empty. As a result, `vy_gc_run()` doesn't remove information about the run from the vylog, which leads to the vinyl/layout test failure.

`vy_run_remove_files()` should return 0 in this case.

Reply:
Fixed.